mtb-ce-manifest-fv2.xml

```diff
@@ -1978,6 +2018,14 @@ iBeacon UUID data.
     <br><br>For more details, see the <a href="https://github.com/Infineon/mtb-example-psoc6-smartio-i2s/blob/master/README.md">README on GitHub</a>.]]></description>
     <req_capabilities>psoc6 smart_io</req_capabilities>
```

Comment:
Should this have been copied/migrated to the req_capabilities_v2 field?

Reply:
@ifyall ,
I see it is already added, can you please check again?